src/overview/onboarding/actions.ts

```diff
@@ -72,18 +74,14 @@ export const fetchShowOnboarding = () => async dispatch => {
     dispatch(setShowOnboardingBox(showOnboardingBox))
 }
 
-/**
- * Sets Power Search Stage as done.
- * Dispatched from the last tooltip in Power Search Stage.
- */
 export const setPowerSearchDone = () => async dispatch => {
-    await setLocalStorage(STORAGE_KEYS.onboardingDemo.step2, 'DONE')
+    await utils.setOnboardingStage('powerSearch', STAGES.done)
```

Comment:
I can see the `'powerSearch'` stage being set in lots of places using that string literal, along with stages like: annotation, tagging, backup. Does it make more sense to put those as part of the constants too?

Reply:
Makes more sense!